cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/OperatorsTest.scala

```diff
@@ -293,27 +292,6 @@ class OperatorsTest extends DocumentingTest with QueryStatisticsTestSupport {
         }
       }
     }
-    section("String operators", "query-operators-string") {
-      p(
-        """The string operators comprise:
-          |
-          |* concatenating strings: `+`
-          |* matching a regular expression: `=~`""".stripMargin)
-      section("Using a regular expression with `=~` to filter words", "syntax-using-a-regular-expression-to-filter-words") {
```

Comment:
I think you should just move this section up instead of editing `comparison.asciidoc`.

Reply:
For some reason, if I add the example in `OperatorsTest.scala`, it breaks the build for neo4j-documentation... that's why I went for `comparison.asciidoc` as the other similar examples are located there.

Reply:
Well the `STARTS WITH` example work just fine in `OperatorsTest`. You just need to get all the opening and closing braces right, I'm sure it will work :smile:

Reply:
Cool. I think I managed to fix this now. Could you please take a last look @inmost-light ?